bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -1,7 +1,17 @@
 VCS_TIMEOUT = 1000ms
 
-$(LINT_DIR)/testbench.sv $(LINT_DIR)/wrapper.sv $(LINT_DIR)/test_bp.sv:
+export SIM_SYNTH_VIVADO_P ?= 0
+export SIM_SYNTH_DC_P ?= 0
+
+$(LINT_DIR)/testbench.sv $(LINT_DIR)/test_bp.sv:
+	@sed "s/BP_CFG_FLOWVAR/$(CFG)/g" $(TB_PATH)/$(TB)/$(@F) > $@
```

Comment:
lint shouldn't have a testbench target

Reply:
Actually, revert all these LINT changes, we're not going to lint the post_synth netlist

Reply:
lint had the testbench target from before. Removed the other stuff though